tests/ui/dyn-star/param-env-infer.rs

```diff
@@ -1,7 +1,10 @@
 // revisions: current next
-//[next] compile-flags: -Ztrait-solver=next
 // incremental
 
+//[next] compile-flags: -Ztrait-solver=next
```

Comment:
This ends up ICEing like:

https://gist.github.com/compiler-errors/2660fdd9902ec2686f6288b697953294

Not sure why it's overflowing now, though. I'll look into it tomorrow.